src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/Models/OrganizationUserInviteDto.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bit.Core.Enums;
+using Bit.Core.Models.Data;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers.Models;
+
+public class OrganizationUserInviteDto
+{
+    public string Email { get; private init; } = string.Empty;
+    public CollectionAccessSelection[] AccessibleCollections { get; private init; } = [];
```

Comment:
nit: collections can also be accessible via groups, which would not appear in this list (which contains direct assignments only).

`AssignedCollections` would be more accurate.